src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -165,7 +166,7 @@ private static void getAllFileInfo(RepoConfiguration config, Path directory, Lis
         try (Stream<Path> pathStream = Files.list(directory)) {
             for (Path filePath : pathStream.collect(Collectors.toList())) {
                 String relativePath = filePath.toString().substring(config.getRepoRoot().length());
-                if (Files.isDirectory(filePath)) {
+                if (Files.isDirectory(filePath) && !relativePath.equals(GIT_DIRECTORY)) {
                     getAllFileInfo(config, filePath, fileInfos);
```

Comment:
It is necessary to traverse into .git directory?

The code here does not really follow the let's directive. `Let's fix this issue by excluding .git directory from authorship generation.`

Reply:
.git directory is included in `Files.list(directory)`, so I guess we cannot avoid not traversing it?

Reply:
I believe skipping it entirely is more representative to the let's directive and this approach is more robust.

```
if (!relativePath.equals(GIT_DIRECTORY)) {
    continue;
}
```

Reply:
>I believe skipping it entirely is more representative to the let's directive and this approach is more robust.

There is still a need to check if its a directory as well (maybe someone wants to create a `.git` file for some reason)

So should be:
```
if (Files.isDirectory(filePath) && relativePath.equals(GIT_DIRECTORY)) {
    continue;
}
```
instead?